ruby_programming/computer_science/lesson_a_very_brief_intro_to_cs.md

```diff
@@ -31,3 +31,11 @@ This section contains helpful links to other content. It isn't required, so cons
 * [Wikipedia on Computer Science](http://en.wikipedia.org/wiki/Computer_science)
 * [Wikipedia on Algorithms](http://en.wikipedia.org/wiki/Algorithm)
 * [Map of Computer Science](https://youtu.be/SzJ46YA_RaA)
+
+### Knowledge Check
+This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
+
+* <a class="knowledge-check-link" href="http://ed.ted.com/lessons/your-brain-can-solve-algorithms-david-j-malan">Write an Algorithm to count the number of people in the room?</a>
```

Comment:
I don't think there actually is a good question to come from this resource, I think it would be best to just remove it.